collectors/organizations.go

```diff
@@ -45,7 +46,7 @@ func NewOrganizationsCollector(
 			Help:        "Labeled Cloud Foundry Organization information with a constant '1' value.",
 			ConstLabels: prometheus.Labels{"environment": environment, "deployment": deployment},
 		},
-		[]string{"organization_id", "organization_name", "quota_name"},
+		[]string{"organization_id", "organization_name", "quota_name", "suspended"},
```

Comment:
@fredga Can you update the cf-cli dependency as well, like last time.

Reply:
Hi,
cf-cli dependency updated.
Regards

Reply:
Thanks :)